src/main/webapp/cleanup.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  <p>
+    <b>Minimal cleanup</b> - Do as minimal workspace changes as possible. Only changed files will be reverted. Private and ignored files will be preserved. This can cause unwanted side effects. It is recommended to explicitly mark files as ignored if those should be kept between builds.
```

Comment:
I'd prefer this long HTML lines to be cut off so they don't exceed the 120-char limit. We prefer to be able to read the code without horizontal scroll or line wrapping 😊

Reply:
Here is something that I do not fully agree. It's XXI century and IDE's and text editors has the functionality of wrapping long lines. But still I understand your point. Changing this.

Reply:
Cool, thanks! As you said, this is not about working in an IDE but rather about displaying code in the GitHub website or diffing changes.